libs/designsystem/src/lib/scss/print/components/_virtual-scroll.scss

```diff
@@ -0,0 +1,5 @@
+@media print {
```

Comment:
I think we should consider renaming this file to something along the lines of `_list.scss` or `_list-virtual-scroll.scss` as in the end it exists to modify the list.